src/s390x/dasd.rs

```diff
@@ -181,7 +181,9 @@ fn is_invalid(dasd: &str) -> Result<bool> {
     let mut cmd = Command::new("fdasd");
     // we're looking for a hardcoded string in the output
     cmd.env("LC_ALL", "C").arg("-p").arg(dasd);
-    Ok(cmd_output(&mut cmd)?.contains("disk label block is invalid"))
+    let invalid = cmd_output(&mut cmd)?.contains("disk label block is invalid");
+    udev_settle()?;
```

Comment:
Can we also add a comment explaining why this is needed?

Reply:
Done